sample/src/main/java/org/hawkular/accounts/sample/control/SetupOperations.java

```diff
@@ -18,24 +18,26 @@
 
 import org.hawkular.accounts.api.OperationService;
 
-import javax.annotation.PostConstruct;
-import javax.ejb.Singleton;
-import javax.ejb.Startup;
+import javax.annotation.Resource;
 import javax.inject.Inject;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.transaction.SystemException;
+import javax.transaction.UserTransaction;
 
 /**
  * @author Juraci Paixão Kröhling
  */
-@Startup
-@Singleton
-public class SetupOperations {
+public class SetupOperations implements ServletContextListener {
```

Comment:
This change is not relevant to the main issue being fixed on this JIRA/PR, but without it, testing with the sample application was causing intermittent problems.